### PR TITLE
Add resource requests to Triage

### PR DESCRIPTION
**Pull request by michaelkolber**
This should make the results more consistent
ref: #18726
/cc BenTheElder spiffxp

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/test-infra-periodics.yaml [hunk 1]
```diff
@@ -45,6 +45,13 @@ periodics:
       - name: service
         mountPath: /etc/service-account
         readOnly: true
+      resources:
+        requests:
+          cpu: 4
```
Comment (BenTheElder) [inline config/jobs/kubernetes/test-infra/test-infra-periodics.yaml:50]: we should set the number of workers flag

Comment (michaelkolber) [reply config/jobs/kubernetes/test-infra/test-infra-periodics.yaml:50]: Why not let it pick it up automatically?

Comment (BenTheElder) [reply config/jobs/kubernetes/test-infra/test-infra-periodics.yaml:50]: Is it correctly detecting it?
There are various mechanisms for finding this info on linux, and IIRC not all of them are correctly masked by containers.

Comment (michaelkolber) [reply config/jobs/kubernetes/test-infra/test-infra-periodics.yaml:50]: I can add some logging so we can be sure

**Pull request merged**